docs/get-started-dapr-actor.md

```diff
@@ -43,7 +43,7 @@ dotnet new classlib -o MyActor.Interfaces
 cd MyActor.Interfaces
 
 # Add Dapr.Actors nuget package
-dotnet add package Dapr.Actors
+dotnet add package Dapr.Actors -v 0.7.0-preview01
```

Comment:
let it pull the latest one rather than a specific version.

Reply:
As long as the nuget packages are prerelease versions, getting the latest version does not work. It does not add a package at all.  
Since there is no `-AllowPrerelease` switch like in the package manager `Install-Package` command, the only way to make sure it 'just works' is specifying the version

Reply:
dotnet feature, targeting 5.0 release: https://github.com/NuGet/Home/issues/4699